Framework/Crystal/inc/MantidCrystal/CentroidPeaks.h

```diff
@@ -38,6 +38,7 @@ class DLLExport CentroidPeaks : public API::Algorithm {
   void integrate();
   void integrateEvent();
   int findPixelID(std::string bankName, int col, int row);
+  void removeEdgePeaks(Mantid::DataObjects::PeaksWorkspace *peakWS);
```

Comment:
I would use `removeEdgePeaks(Mantid::DataObjects::PeaksWorkspace &peakWS);` instead and call it as `removeEdgePeaks(*ws)`. That way `removeEdgePeaks` doesn't need to check for a null pointer.